packages/web/src/components/organisms/AssetOutline/index.tsx

```diff
@@ -1,47 +1,65 @@
-import React from 'react'
-import { List, Space } from 'antd'
+import React, { useMemo } from 'react'
+import { List, Space, Row, Col } from 'antd'
 import { CircleGraph } from 'src/components/atoms/CircleGraph'
 import { useAssetStrength } from 'src/fixtures/dev-kit/hooks'
-import { truncate } from 'src/fixtures/utility/string'
+import { useGetPropertyAuthenticationQuery } from '@dev/graphql'
 
 interface Props {
-  metricsAddress: string
-  marketAddress: string
+  propertyAddress: string
 }
 
-export const AssetOutline = ({ metricsAddress, marketAddress }: Props) => {
-  const { assetStrength } = useAssetStrength(metricsAddress, marketAddress)
-  const includedAssets = ['x-lib', 'x-plugin-lib', 'x-xxx', 'x-xxxxxxxxxxxxxxxxxxxx']
+const AssetStrengthBase = ({ assetStrength }: { assetStrength: number }) => (
+  <Space direction="horizontal" size={40}>
+    <CircleGraph size={224} percentage={assetStrength} />
+    <div>
+      <span style={{ fontSize: '36px', lineHeight: '48px', color: '#000' }}>{Math.round(assetStrength * 100)}%</span>
+      <span> of total market</span>
+    </div>
+  </Space>
+)
+
+const AssetStrength = ({ metrics, market }: { metrics: string; market: string }) => {
+  const { assetStrength: maybeAssetStrength } = useAssetStrength(metrics, market)
+  const assetStrength = useMemo(() => maybeAssetStrength || 0, [maybeAssetStrength])
+  return <AssetStrengthBase assetStrength={assetStrength} />
+}
+
+const AssetStrengthWithoutData = () => {
+  return <AssetStrengthBase assetStrength={0} />
+}
```

Comment:
I subdivided Components so that I don't have to deal with undefined in hooks. Let me know if we need to separate the files.